project.godot

```diff
@@ -11,7 +11,7 @@ config_version=5
 [application]
 
 config/name="LudumDare53"
-run/main_scene="res://levels/temp_catcher_level.tscn"
```

Comment:
Probably want to discard this change and leave your temp level as the default scene